packages/react/dismissable-layer/src/DismissableLayer.tsx

```diff
@@ -219,7 +220,10 @@ type FocusOutsideEvent = CustomEvent<{ originalEvent: FocusEvent }>;
  * to mimic layer dismissing behaviour present in OS.
  * Returns props to pass to the node we want to check for outside events.
  */
-function usePointerDownOutside(onPointerDownOutside?: (event: PointerDownOutsideEvent) => void) {
+function usePointerDownOutside(
+  ownerDocument: Document,
+  onPointerDownOutside?: (event: PointerDownOutsideEvent) => void
+) {
```

Comment:
For consistency, could we follow the same API as you did in `onEscapeKeydown` where the `ownerDocument` comes second and default to `document`?

Reply:
Done.